content/en/docs/reference/glossary/priority-class.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Priority Class
+id: priority-class
+date: 2023-07-10
+full_link: /docs/concepts/configuration/pod-priority-preemption/#priorityclass
```

Comment:
Looks like this should actually be `docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass`?

Reply:
Sure! I will make these changes

Reply:
👋 @EraKin575 

Can you update the `full_link` field?

Reply:
Yes I have. Thanks for the feedback. Please tell me if further changes  are required